ft_calloc.c

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_calloc.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: kkubik <kkubik@student.42prague.com>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/01/12 22:55:23 by kkubik            #+#    #+#             */
+/*   Updated: 2023/01/12 23:06:47 by kkubik           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stddef.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+void	*ft_calloc(size_t nitems, size_t size);
+
```

Comment:
needs to change name of function to ft_calloc

Reply:
Thank you for noticing! it would be the worst mistake ever to send it like this T_T